[22/Sep/2019 23:17:14] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:15] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:17] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:40] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:42] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:43] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:45] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:46] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:48] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:17:49] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:06] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:08] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:09] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:11] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:16] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:18] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:20] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:25] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:30] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:35] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:54] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:18:57] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:02] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:25] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:27] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:31] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:33] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:43] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:19:59] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:20:01] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:20:08] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:22] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:23] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:24] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:25] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:25] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:26] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:27] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:28] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:29] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:29] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:30] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:31] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:32] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:40] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:41] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:42] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:43] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:43] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:44] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:44] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:46] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:47] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:49] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:50] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:50] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:51] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:22:52] "GET / HTTP/1.1" 200 240
Error: That port is already in use.
[22/Sep/2019 23:55:25] "GET / HTTP/1.1" 200 240
Not Found: /favicon.ico
[22/Sep/2019 23:55:25] "GET /favicon.ico HTTP/1.1" 404 2304
[22/Sep/2019 23:55:36] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:55:57] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:55:58] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:55:59] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:56:17] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:56:20] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:56:21] "GET / HTTP/1.1" 200 240
[22/Sep/2019 23:56:22] "GET / HTTP/1.1" 200 240
[23/Sep/2019 01:50:08] code 400, message Bad request syntax ('\x00\x00\x00TZ\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x04\x010I\x00\x00\x00\x00\x80\x0b\xa8\xc0\x00\x0c)t F\x07\x00tsXrcsXYs9\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00bbXcsXctst\x00\x00\x00\x00\x00\x00')
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 868, in emit
    msg = self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 741, in format
    return fmt.format(record)
  File "/home/brettnorton7/.local/lib/python2.7/site-packages/django/utils/log.py", line 192, in format
    return super(ServerFormatter, self).format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 476, in format
    raise e
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 25: ordinal not in range(128)
Logged from file basehttp.py, line 124
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 02:05:05] "GET / HTTP/1.0" 400 61185
[23/Sep/2019 02:41:57] "GET / HTTP/1.1" 200 240
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 03:24:01] "GET / HTTP/1.0" 400 61183
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 03:24:17] "GET / HTTP/1.0" 400 61183
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 03:45:55] "GET / HTTP/1.0" 400 61185
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 06:50:00] "GET / HTTP/1.0" 400 61184
[23/Sep/2019 07:16:21] code 400, message Bad HTTP/0.9 request type ("\x16\x03\x01\x00u\x01\x00\x00q\x03\x03\xf7\x1c\xbd\x84\xe4h\xf9\xf8\x1f\xee~\xcd\x12\xa2'\xfd\x98\xd7\xc5\xad\xe5i\xd2\x07>\xafu\xc8\x13\xf9\xa2\xda\x00\x00\x1a\xc0/\xc0+\xc0\x11\xc0\x07\xc0\x13\xc0")
[23/Sep/2019 07:16:21] You're accessing the development server over HTTPS, but it only supports HTTP.

Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 08:39:44] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 09:43:17] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 10:01:39] "GET / HTTP/1.0" 400 61183
[23/Sep/2019 17:05:11] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:24] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:25] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:25] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:26] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:27] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:05:28] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:23:00] "GET / HTTP/1.1" 200 240
Not Found: /favicon.ico
[23/Sep/2019 17:23:00] "GET /favicon.ico HTTP/1.1" 404 2304
[23/Sep/2019 17:23:02] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:23:02] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:23:23] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:28:03] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:28:18] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:28:25] "GET / HTTP/1.1" 200 240
[23/Sep/2019 17:28:46] "GET / HTTP/1.1" 200 240
Not Found: /favicon.ico
[23/Sep/2019 18:30:39] "GET /favicon.ico HTTP/1.1" 404 2304
[23/Sep/2019 18:30:41] "GET / HTTP/1.1" 200 240
[23/Sep/2019 18:30:42] "GET / HTTP/1.1" 200 240
[23/Sep/2019 18:30:44] "GET / HTTP/1.1" 200 240
[23/Sep/2019 18:31:04] "GET / HTTP/1.1" 200 240
[23/Sep/2019 18:31:07] "GET / HTTP/1.1" 200 240
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 19:08:48] "GET / HTTP/1.0" 400 61185
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 22:26:43] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 22:27:20] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 22:27:28] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 22:27:43] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[23/Sep/2019 22:35:35] "GET / HTTP/1.0" 400 61184
[24/Sep/2019 00:30:40] code 400, message Bad request version ('HTTP')
[24/Sep/2019 00:30:40] "GET ../../mnt/custom/ProductDefinition HTTP" 400 -
[24/Sep/2019 00:45:50] code 400, message Bad request syntax ('JDWP-Handshake\\0\\0\\0\\11\\0\\0\\0\\1\\0\\1\\1')
[24/Sep/2019 00:45:50] "JDWP-Handshake\0\0\0\11\0\0\0\1\0\1\1" 400 -
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 06:38:30] "GET / HTTP/1.0" 400 61185
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 06:57:45] "GET / HTTP/1.0" 400 61185
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 08:34:42] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 08:35:01] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 13:35:01] "GET / HTTP/1.0" 400 61183
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 13:54:12] "GET / HTTP/1.0" 400 61185
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 596, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 331, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 652, in __init__
    self.handle()
  File "/home/brettnorton7/.local/lib/python2.7/site-packages/django/core/servers/basehttp.py", line 140, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python2.7/socket.py", line 480, in readline
    data = self._sock.recv(self._rbufsize)
error: [Errno 104] Connection reset by peer
[24/Sep/2019 18:46:03] "GET / HTTP/1.1" 200 240
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 19:13:50] "GET / HTTP/1.0" 400 61183
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[24/Sep/2019 20:32:55] "GET / HTTP/1.0" 400 61183
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 01:57:55] "GET / HTTP/1.0" 400 61184
[25/Sep/2019 03:26:37] "GET / HTTP/1.1" 200 240
[25/Sep/2019 06:30:46] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*\xe0\x00\x00\x00\x00\x00Cookie:')
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 868, in emit
    msg = self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 741, in format
    return fmt.format(record)
  File "/home/brettnorton7/.local/lib/python2.7/site-packages/django/utils/log.py", line 192, in format
    return super(ServerFormatter, self).format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 476, in format
    raise e
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe0 in position 6: ordinal not in range(128)
Logged from file basehttp.py, line 124
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 07:22:16] "GET / HTTP/1.0" 400 61185
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 08:18:04] "GET / HTTP/1.0" 400 61181
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 09:33:23] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 10:39:31] "GET / HTTP/1.0" 400 61185
[25/Sep/2019 10:49:08] code 400, message Bad request version ('HTTP')
[25/Sep/2019 10:49:08] "GET ../../mnt/custom/ProductDefinition HTTP" 400 -
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 11:47:35] "GET / HTTP/1.0" 400 61184
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 14:42:13] "GET / HTTP/1.0" 400 61182
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 14:48:06] "GET / HTTP/1.0" 400 61182
Invalid HTTP_HOST header: u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal:8000'. You may need to add u'python-virtual-machine.us-central1-a.c.software-engineering-251917.internal' to ALLOWED_HOSTS.
[25/Sep/2019 17:12:45] "GET / HTTP/1.0" 400 61184
